exp-player/addon/components/exp-free-response/component.js

```diff
@@ -48,21 +47,21 @@ export default ExpFrameBaseComponent.extend(Validations, {
       return label;
     }),
     diff1: Ember.computed('q1', function() {
-        var remaining = getRemaining(this.get('q1'));
+        var length = getLength(this.get('q1'));
```

Comment:
The default value is `undefined` so it will complain if I try to do something like `Ember.get(this.get('q1'), 'length')`.